Description: Use Debian gfan package instead of gfan shipped with M2.
Author: Doug Torrance <dtorrance@piedmont.edu>
Last-Update: 2020-05-05

--- a/M2/Macaulay2/packages/gfanInterface.m2
+++ b/M2/Macaulay2/packages/gfanInterface.m2
@@ -13,7 +13,7 @@
 	    {Name => "Diane Maclagan (current maintainer)", Email => "D.Maclagan@warwick.ac.uk", HomePage=>"http://homepages.warwick.ac.uk/staff/D.Maclagan/"}},
 	Headline => "interface to Anders Jensen's Gfan software",
 	Configuration => {
-		"path" => "",
+		"path" => "/usr/bin/",
 		"fig2devpath" => "",
 		"keepfiles" => true,
 		"verbose" => false,
--- a/M2/Macaulay2/packages/StatePolytope.m2
+++ b/M2/Macaulay2/packages/StatePolytope.m2
@@ -9,7 +9,7 @@
     	Headline => "the state polytope of an ideal",
 	-- DebuggingMode should be true while developing a package, 
 	--   but false after it is done
-	Configuration => {"gfan command" => "gfan"},
+	Configuration => {"gfan command" => "/usr/bin/gfan"},
     	DebuggingMode => false,
 	AuxiliaryFiles => true,
 	CacheExampleOutput => true
